src/java.base/share/classes/java/io/DataInput.java

```diff
@@ -493,7 +493,7 @@
      * is encountered, it is discarded and reading
      * ceases. If the character {@code '\r'}
      * is encountered, it is discarded and, if
-     * the following byte converts &#32;to the
+     * the following byte converts to the
```

Comment:
The above is a bit confusing as it reads(same in ImageInputStream.java).  I think that that can be looked at later.

Reply:
> The above is a bit confusing as it reads(same in ImageInputStream.java). I think that that can be looked at later.

Just to clarify: you are okay with removing the `&#32;` entity, right? As for ImageInputStream, some doc comments should probably use `@inheritDoc`. But this is a much bigger clean-up.

Reply:
> > The above is a bit confusing as it reads(same in ImageInputStream.java). I think that that can be looked at later.
> 
> Just to clarify: you are okay with removing the `&#32;` entity, right? As for ImageInputStream, some doc comments should probably use `@inheritDoc`. But this is a much bigger clean-up.

Yes I am.  It just does not flow as well as it could IMHO